pootle/apps/pootle_app/management/commands/initdb.py

```diff
@@ -33,7 +33,7 @@ class Command(NoArgsCommand):
 
     def handle_noargs(self, **options):
         self.stdout.write('Populating the database.')
-        initdb(options["create_projects"])
+        InitDB().init(options["create_projects"])
```

Comment:
no - i just changed the name of the method - i didnt want to trigger the db install on object creation for several reasons